packages/posthog/src/web.ts

```diff
@@ -80,9 +86,66 @@ export class PosthogWeb extends WebPlugin implements PosthogPlugin {
 
   async setup(options: SetupOptions): Promise<void> {
     const host = options.host || 'https://us.i.posthog.com';
-    posthog.init(options.apiKey, {
+    const config: any = {
       api_host: host,
-    });
+    };
+
+    // Check for Capacitor configuration
+    const capacitorConfig = (window as any).Capacitor?.getConfig?.()?.plugins
```

Comment:
Capacitor Configuration file is officially only supported on Android and iOS. Please remove this workaround and just add the new properties to the `SetupOptions` interface.